spec/datadog/tracing/trace_operation_spec.rb

```diff
@@ -88,6 +88,68 @@
       it do
         expect(trace_op.send(:metrics)).to eq({})
       end
+
+      context 'when 128 bit trace id generation enabled' do
+        around do |example|
+          ClimateControl.modify(
+            'DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED' => 'true'
```

Comment:
You should mock `allow(Datadog.configuration.tracing).to receive(:trace_id_128_bit_generation_enabled).and_return(true)` here, given we don't actually depend on the environment variable, and the environment variable test is already done by the original PR that introduced `Datadog.configuration.tracing.trace_id_128_bit_generation_enabled`.